runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsPlugin.java

```diff
@@ -206,6 +217,16 @@ public void onItemSpawned(ItemSpawned itemSpawned)
 		{
 			existing.setQuantity(existing.getQuantity() + groundItem.getQuantity());
 		}
+
+		if (!config.notifierThreshold().equals(ValueThreshold.OFF))
```

Comment:
Reversing the condition and simply returning early would make this easier to understand and save on nesting `if`'s